### PR TITLE
[BEAM-1251] Migrate away from xrange()

### PR DESCRIPTION
**Pull request by cclauss**
__xrange()__ was removed from Python 3 in favor of an iterator-based __range()__.

For all files that use the Python 2-only builtin function __xrange()__, add the line __from six.moves import xrange__ for compatibility with both Python 2 and Python 3.

### CODE REVIEW

**Comment by robertwb** [comment]
It seems for most, if not all, of these uses of xrange, range would be perfectly acceptable as well (i.e. the ranges in question are quite small). Could you change these to use the vanilla range rather than import xrange from six?

**Comment by cclauss** [comment]
The lifetime of Python 2 is now just two years long.  An alternative (probably safer) approach would be to leave this PRs as written until 2020 and then dropping Python 2 support will only require 1) removing the import and 2) doing a global find-and-replace of 'xrange(' --> 'range('.

I am willing to go whichever way you feel is best.

**Comment by robertwb** [comment]
Unless there is a compelling reason to use xrange over range for any of these in Python 2 (I don't see one), let's change to range now.

**Comment by cclauss** [comment]
I could not see any of these that I felt _needed_ to be iterators in Python 2.  Please let me know is you see any that could get massive.

**Comment by robertwb** [comment]
All look good to me. Will merge once all tests pass (not that I'm worried).

File: sdks/python/apache_beam/transforms/trigger_test.py [hunk 1]
```diff
@@ -21,7 +21,6 @@
 import os.path
 import pickle
 import unittest
-
```
Comment (robertwb) [inline sdks/python/apache_beam/transforms/trigger_test.py]: Lint seems to want this newline.

**Comment by cclauss** [comment]
@robertwb Is there something more that I need to do on this PR?

**Pull request merged**